cdc/split.hh

```diff
@@ -66,12 +66,14 @@ public:
     // Tells processor we have reached end of record - last part
     // of a given timestamp batch
     virtual void end_record() = 0;
+
+    virtual row_states_map& clustering_row_states() = 0;
```

Comment:
Not very happy that we are exposing internals of the change processor like that. I'm not sure we have enough time to think of a better way, so I am OK with incurring this technical debt.

Perhaps, at least, we can make the reference (and the method) const?